.husky/email-check.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# This script will check for, and reject, non-github email addresses
+
+# Get user email used in git config
+EMAIL=$(git config user.email)
+
+# Regex for a private github email address
+MATCH='[A-Za-z0-9._%+-]+@users\.noreply\.github\.com$'
```

Comment:
Personally I'm not a fan of this hook, because if you have a github account anonymizing your email, it won't work.

Reply:
@GCHQ-Developer-530 Can I remove this? I always commit with `--no-verify` ic-ui-kit. Dont really see the point of checking for github no reply email, an account is already linked to an email anyways

Reply:
It needs to stay as an extra check for us

Reply:
Ok